### DISCUSSION TITLE
Registries and bootstrap

### DISCUSSION THREAD

**Discussion by asfernandes** [microsoft/vcpkg#28424] [Q&A]
I do want to use vcpkg in manifest mode and using a custom registry together with the default (vcpkg) registry.

I see I could point the vcpkg github as `default-registry`.

In this case we don't need to checkout whole vcpkg repository for the bootstrap?

Can we have it working only having `vcpkg.exe`?

**Comment by Osyotr** [comment]
No, you also need stuff from scripts directory, probably something else.
